.github/ISSUE_TEMPLATE/feature-request.yml

```diff
@@ -1,9 +1,11 @@
-name: "💡 Feature Request"
-description: Create a ticket requesting a new feature
-title: "💡 [PLUGIN] - <title>"
+name: Feature Request
+description: Suggest an idea for this project
+title: "[name of the plugin e.g. kube-monitoring] Issue Title"
 labels: [
   "plugin"
 ]
+assignees:
+  - 
```

Comment:
empty list here, should we maybe comment this out? 
or we some information who can be put in here?
actually this should be assigned by us, I'd say.

Reply:
This actually prevents you from assigning someone. This should always be done by us.